features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -193,7 +194,7 @@ export function SidebarNavigation() {
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              onClick={() => sendEmail()}
```

Comment:
This is a bit of a trick task 😄 If you had a link here instead of a button you could simply set the `href` to `mailto:support@prolog-app.com?subject=Support Request`. This is a bit more standard and also easier to test. Feel free to add a Cypress test btw.